CondCore/SiPixelPlugins/BuildFile.xml

```diff
@@ -3,4 +3,3 @@
 <use name="CondCore/ESSources"/>
 <use name="CondFormats/SiPixelObjects"/>
 <use name="CondFormats/DataRecord"/>
-<flags EDM_PLUGIN="1"/>
```

Comment:
This package does not define any librares, only plugins (in the `plugins/` subdiectory).
Maybe this BuildFile.xml could be completely removed ?

Reply:
Yeah I was considering that, but went for minimal changes in the end. Now that you suggested it too I'll do it.

Reply:
Oh wait, I remember why I didn't do it! This package is problematic anyway, because in the `test`, it includes most files from the `plugins` directly. So it seemed that it might on the long term be more appropriate to maybe actually introduce a library that is then used both by `plugins` and `test`. But since these changes would be more involved and are not in the scope of BuildFile cleaning PRs.

Reply:
In my newest commit I addressed the other two comments but not this one. I'm afraid the issue that also triggered your comment is too big for this PR, because one should also consider the relation to the `test` code.